src/zope/component/interfaces.py

```diff
@@ -364,6 +364,12 @@ def getInterfaces():
 
 def _inherits_docs(func, iface):
     doc = iface[func.__name__].__doc__
+
+    # doc can be None if the interpreter drops all docstrings when the
+    # environment variable PYTHONOPTIMIZE is set 2.
+    if not doc:
```

Comment:
Tiny bit nitpicky: This also matches the empty string, and `is None` should be cheaper because it boils down to pointer comparison.